rtos/source/ThisThread.cpp

```diff
@@ -251,4 +251,32 @@ const char *get_name()
 #endif
 }
 
+osPriority_t ThisThread::get_priority()
+{
+#if MBED_CONF_RTOS_PRESENT
+    osThreadId_t id = osThreadGetId();
+    if (id == nullptr) {
+        return osPriorityError;
+    }
+
+    return osThreadGetPriority(id);
+#else
+    return (osPriorityError)(-1);
```

Comment:
The definition of `osPriorityError` might not be available if `MBED_CONF_RTOS_PRESENT ` is not present.

Reply:
Fixed